src/Microsoft.Azure.WebJobs.Extensions.OpenApi.Core/SwaggerUI.cs

```diff
@@ -88,9 +84,7 @@ public ISwaggerUI AddServer(IHttpRequestDataObject req, string routePrefix, IOpe
             }
 
             this._baseUrl = servers.First().Url;
-
-            absolutePath = new Uri(this._baseUrl).AbsolutePath.TrimEnd('/');
-            this._swaggerUiApiPrefix = absolutePath;
+            this._swaggerUiApiPrefix = this._baseUrl.TrimEnd('/');
```

Comment:
Let's keep the existing codebase for this line. Rather, the live above is the issue, if one add a server URL like `/api`, instead of `https://my.domain.com/xxxx`.

Reply:
I'll take your change.